docs/start/usage/web-app.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
Again, this is not the use case of WasmEdge. It tells users how to create a docker image for users. You can discard this chapter. We have the same file on https://github.com/second-state/wasmedge-containers-examples/blob/main/simple_wasi_app.md

Reply:
Gotcha